README.md

```diff
@@ -1,3 +1,39 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Market Overview
+
+> Market Overview is a website designed to showcase the US stock market. You could find details about the stock of choice and improve your strategies. Built using React, Redux, SASS, and tested using React testing library.
+
+## Built With
+
+- React, Redux, SASS
+- Visual Studio Code.
```

Comment:
- [ ] **Good job Carlos 👏🎉🥳** Please add the screenshot of the app to make the readme professional 🙂

Reply:
Absolutely! thanks for the suggestion @Rustamxon7!